README.md

```diff
@@ -30,11 +30,9 @@ The code snippet below illustrates how to make a simple GET request to a remote
 ```swift
 import AsyncHTTPClient
 
-let httpClient = HTTPClient(eventLoopGroupProvider: .singleton)
-
 /// MARK: - Using Swift Concurrency
 let request = HTTPClientRequest(url: "https://apple.com/")
-let response = try await httpClient.execute(request, timeout: .seconds(30))
+let response = try await HTTPClient.shared.execute(request, timeout: .seconds(30))
```

Comment:
@Lukasa & @glbrntt Here I change all of them from `HTTPClient.init` + `HTTPClient.shutdown` --> `HTTPClient.shared`